maven-extension/src/main/java/io/opentelemetry/maven/OpenTelemetrySdkService.java

```diff
@@ -80,7 +80,7 @@ public OpenTelemetrySdkService() {
         configProperties.getBoolean("otel.instrumentation.maven.mojo.enabled", true);
     // TODO maybe default would be false
     this.transferInstrumentationEnabled =
-        configProperties.getBoolean("otel.instrumentation.maven.transfer.enabled", true);
+        configProperties.getBoolean("otel.instrumentation.maven.transfer.enabled", false);
```

Comment:
Disable by default. I'm wondering if we could add later the option to create spans if the duration of a transfer is beyond a given threshold